app/models/osf-model.ts

```diff
@@ -20,16 +19,17 @@ import authenticatedAJAX from 'ember-osf-web/utils/ajax-helpers';
  * @class OsfModel
  * @public
  */
-export default DS.Model.extend({
+
+const OsfModel = DS.Model.extend({
     links: DS.attr('links'),
     embeds: DS.attr('embed'),
 
     relationshipLinks: alias('links.relationships'),
-    _dirtyRelationships: null,
-    isNewOrDirty() {
+    isNewOrDirty(): boolean | number {
```

Comment:
Given the name, it'd make more sense to use just `boolean`, and do `!!Object.keys(...).length` (or `Boolean(...)`) to make that true.